app/controllers/api/v1/monsters_controller.rb

```diff
@@ -1,7 +1,8 @@
 class Api::V1::MonstersController < ApplicationController
 
   def index 
-    render json: Monster.all
+    user_status = current_user()
+    render json: {monsters: Monster.all, user: user_status}
```

Comment:
We can dry this up. Lets update to: 
```
 render json: {monsters: Monster.all, user: current_user}
```
You'll also want to avoid using `()` for ruby methods. There is no functional downside, its just not convention